src/actions/l10n.js

```diff
@@ -40,45 +36,9 @@ export function receiveL10n(
   };
 }
 
-/**
- * This function is called when the AppLocalizationProvider is mounted.
- * It takes the locales available and generates l10n bundles for those locales.
- * Initially it dispatches the info that translations are now fetching and
- * later it dispatches the translations and updates the state
- */
-export function setupLocalization(
```

Comment:
This is moved without any change to `src/components/app/AppLocalizationProvider.js`